src/interactivity/win32/UiaTextRange.hpp

```diff
@@ -69,6 +67,8 @@ typedef unsigned int Column;
 // the first char of the 0th row in the text buffer row array.
 typedef unsigned int Endpoint;
 
+constexpr IdType invalid_id = 0;

```

Comment:
@adiviness What is the recommended naming scheme for constants?  I have seen all uppercase, pascal, lowercase, etc...   Which one is correct?

Reply:
It depends on the scope of the constant and what sort of constant it is. For macros we typically do all caps (ex. FOO_BAR). for constexprs we typically do pascal case (ex. FooBar).

Reply:
Relevant ISO C++ guideline:

[ES.9: Avoid ALL_CAPS names](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#es9-avoid-all_caps-names)

TL;DR is that since ALL CAPS names are typically used for macros, it's dangerous to use them for anything but macros since it makes it easy for a macro to silently replace or redefine something that's not a macro.

Reply:
Thanks @binarycrusader - this is why I was asking.  I have seen a lot of constexpr that were upper case.  👎    I noticed an issue was already logged.  👍